### PR TITLE
add beach park, Il File

### PR DESCRIPTION
**Pull request by pjbrown9**
This PR adds the Beach Park file to our directory.
@psaweh could I borrow your :eyes: ?

### CODE REVIEW

File: pjbrown-beach park.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Beack Park. IL
```
Comment (psaweh) [inline pjbrown-beach park.md:1]: Never been there.  Where is Beack Park located.

Comment (psaweh) [inline pjbrown-beach park.md:1]: Hi

**Review by psaweh — APPROVED** [review]

File: pjbrown-beach park.md (hunk 1 above)
Comment (pjbrown9) [reply pjbrown-beach park.md:1]: Northeast of Six Flags.

Comment (pjbrown9) [reply pjbrown-beach park.md:1]: Hi

**Review by psaweh — APPROVED** [review]
Looks good to me

**Pull request merged**